.python-version

```diff
@@ -1 +1 @@
-3.7.0
+3.8.5
```

Comment:
you do not need to submit this file. I think you should remove from git

Reply:
Its there for pyenv users. We can remove it, but also doesn't do any harm to retain.

Reply:
I use pyenv but it might create a conflict and for no pyenv users it does not add anything.

Reply:
ok, lets remove it